pkg/metrics/aggregator.go

```diff
@@ -90,7 +91,6 @@ func NewMetricsAggregator(aggregationInterval time.Duration) *AdoptionMetricsAgg
 		aggregationInterval: aggregationInterval,
 	}
 	collector.clusterAdmin.Set(0)
-	collector.limitedSupport.Set(0)
```

Comment:
Why was this removed?

Reply:
there is no Set method for GaugeVec if I am not mistaken

Reply:
I can not answer that. I will run a test as soon as I sort out the errors.

Reply:
It does seem that we need an initial value, I am looking up in the docs how this can be done.

Reply:
Actually, upon looking closer, I think our code will set it after checking. We might be Ok.